src/infra/dev/variables.tf

```diff
@@ -273,11 +273,92 @@ variable "tags" {
   }
 }
 
-variable "ecs_as_threshold" {
-  type    = number
-  default = 80
+variable "ecs_alarms" {
+  type = map(object({
+    metric_name = string
+    namespace = string
+    threshold = optional(number)
+    evaluation_periods = optional(number)
+    period = optional(number)
+    statistic = optional(string)
+    comparison_operator = optional(string)
+    
+    ok_actions = optional(list(string))
+  }))
+
+  default = {
+    "ecs-cpu-utilization" = {
+      metric_name = "CPUUtilization"
+      namespace   = "AWS/ECS"
+      evaluation_periods = 1
+      comparison_operator = "GreaterThanOrEqualToThreshold"
+      period              = 60
+      statistic           = "Average"
+    },
+    "ecs-memory-utilization" = {
+      metric_name = "MemoryUtilization"
+      namespace   = "AWS/ECS"
+      evaluation_periods = 1
+      comparison_operator = "GreaterThanOrEqualToThreshold"
+      period              = 60
+      statistic           = "Average"
+    }
+  }
 }
 
+variable "lambda_alarms" {
+  type = object({
+    metric_name = string
+    namespace = string
+    threshold = optional(number)
+    evaluation_periods = optional(number)
+    period = optional(number)
+    statistic = optional(string)
+    comparison_operator = optional(string)
+    sns_topic_alarm_arn = optional(list(string))
+    ok_actions = optional(list(string))
+  })
+
+  default = {
+    
+      metric_name = "Errors"
+      namespace   = "AWS/Lambda"
+      threshold = 1
+      evaluation_periods = 2
+      comparison_operator = "GreaterThanOrEqualToThreshold"
+      period              = 60
+      statistic           = "Sum"
+      
+    }
+  }
+
+  variable "dlq_alarms" {
```

Comment:
We have to monitor 2 dead letter queue: 
* the lambda one 
* the event bridge one.